ingestion-edge/README.md

```diff
@@ -84,82 +90,86 @@ environment variables:
 
 ## Testing
 
-Run all tests locally with `docker-compose` or
-[CircleCI Local CLI](https://circleci.com/docs/2.0/local-cli/#installing-the-circleci-local-cli-on-macos-and-linux-distros)
+Run tests with [CircleCI Local
+CLI](https://circleci.com/docs/2.0/local-cli/#installing-the-circleci-local-cli-on-macos-and-linux-distros),
+`docker-compose`, or `pytest`
 
 ```bash
-# only print test logs and leave other services running
-docker-compose run --rm test
-
-# rebuild images, recreate containers, stop all containers when one exits, use SIGKILL immediately
-docker-compose up --build --force-recreate --abort-on-container-exit --timeout=0
-
 # circleci
 (cd .. && circleci build --job ingestion-edge)
-```
-
-### Style Checks
 
-Update dependencies installed in the container as needed
+# docker-compose
+docker-compose run --rm test
 
-```bash
-docker-compose build
+# pytest
+./bin/pytest --black --docstyle --flake8 --mypy --mypy-ignore-missing-imports --ignore tests/load
 ```
 
+### Style Checks
+
 Run style checks
 
 ```bash
-docker-compose run --rm --no-deps test --black --docstyle --flake8 --mypy --mypy-ignore-missing-imports ingestion_edge
+# docker-compose
+docker-compose run --rm test --black --docstyle --flake8 --mypy --mypy-ignore-missing-imports ingestion_edge
+
+# pytest
+./bin/pytest --black --docstyle --flake8 --mypy --mypy-ignore-missing-imports ingestion_edge
 ```
 
 ### Unit Tests
 
-Update dependencies installed in the container as needed
-
-```bash
-docker-compose build
-```
-
 Run unit tests
 
 ```bash
-docker-compose run --rm --no-deps test tests/unit
+# docker-compose
+docker-compose run --rm test tests/unit
+
+# pytest
+./bin/pytest tests/unit
 ```
 
 ### Integration Tests
 
-Update dependencies installed in the container as needed
-
-```bash
-docker-compose build
-```
-
```

Comment:
removed comment indicating builds because that's already covered in the build section